WcaOnRails/config/puma.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
Are we going to move from unicorn to puma? I found [this short reading](https://blog.lucascaton.com.br/2016/03/13/puma-vs-unicorn/). @jfly, @larspetrus, @timhabermaas